crates/rpc/rpc-builder/tests/it/http.rs

```diff
@@ -514,7 +515,7 @@ async fn test_call_otterscan_functions_http_and_ws() {
 // <https://github.com/paradigmxyz/reth/issues/5830>
 #[tokio::test(flavor = "multi_thread")]
 async fn test_eth_logs_args() {
-    reth_tracing::init_test_tracing();
+    TestTracer.init().unwrap();
```

Comment:
I'd like to keep the function but we can make this the body of the init_test_tracing function